addOns/pscanrulesAlpha/src/main/resources/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -43,3 +43,45 @@ pscanalpha.sourcecodedisclosure.extrainfo = {0}
 pscanalpha.sourcecodedisclosure.name = Source Code Disclosure
 pscanalpha.sourcecodedisclosure.refs = http://blogs.wsj.com/cio/2013/10/08/adobe-source-code-leak-is-bad-news-for-u-s-government/
 pscanalpha.sourcecodedisclosure.soln = Ensure that application Source Code is not available with alternative extensions, and ensure that source code is not present within other files or data deployed to the web server, or served by the web server. 
+
+pscanalpha.metadata-request-headers.name = Fetch Metadata Request Headers
+
+pscanalpha.metadata-request-headers.sfs.missing.name = Sec-Fetch-Site Header is Missing
+pscanalpha.metadata-request-headers.sfs.missing.desc = Specifies the relationship between request initiator's origin and target's origin.
+pscanalpha.metadata-request-headers.sfs.missing.refs = https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-Fetch-Site
+pscanalpha.metadata-request-headers.sfs.missing.soln = Ensure that Sec-Fetch-Site header is included in request headers.
+
+pscanalpha.metadata-request-headers.sfs.invalid-values.name = Sec-Fetch-Site Header has Invalid Value
```

Comment:
I'd suggest a slight change for the "Invalid" set of names.

> Sec-Fetch-Site Header Has an Invalid Value

https://capitalizemytitle.com/style/Chicago/

Reply:
done